lib/graphql/subscription.ts

```diff
@@ -0,0 +1,34 @@
+import { GraphQLClient } from "./client";
+import {
+  MutationUpdateSubscriptionPlanArgs,
+  UpdateSubscriptionPlanResult,
+} from "./schema";
+
+const UPDATE_SUBSCRIPTION_PLAN = `mutation updateSubscriptionPlan(
+  $newPlan: PurchaseType!
+) {
+  updateSubscriptionPlan(newPlan: $newPlan) {
+    newPlan
+    previousPlans
+    updateActiveAt
+    hasCanceledDowngrade
+    hasOrderedPhysicalCard
+  }
+}`;
+
+export class Subscription {
+  constructor(protected client: GraphQLClient) {}
+
+  /**
+   * Updates subscription plan
+   *
+   * @param args  query parameters including new plan name
+   * @returns     result of update
+   */
+  public async updateSubscriptionPlan(
```

Comment:
what about `updatePlan` or just `update` instead of `updateSubscriptionPlan`?

For most existing methods we don't repeat the model name (e.g. `transaction.categorize()`)

Reply:
agree, fixed in 8e3a79d